lib/nodeMap.js

```diff
@@ -64,7 +64,7 @@ api.createNodeMap = (input, graphs, graph, issuer, name, list) => {
   // add values to list
   if(graphTypes.isValue(input)) {
     if('@type' in input) {
-      const type = input['@type'];
+      let type = input['@type'];
```

Comment:
The fact that this did not trigger when running tests implies a missing test case.

Reply:
Yes. "npm run coverage" is helpful to find such things. It does show that line not being executed.